python-frontend/src/main/java/org/sonar/plugins/python/api/symbols/ClassSymbol.java

```diff
@@ -33,6 +35,8 @@
 
   Set<Symbol> declaredMembers();
 
+  Map<Symbol, InferredType> fieldsWithTypes();
```

Comment:
I'm not so sure about this API `fieldsWithTypes`.
I think I would prefer having an API on Symbol `InferredType declaredType()`

Reply:
Well. We don't need this on anything but fields. Exposing this on symbol level - we will expose this information for functions and classes (probably, later for module symbols as well) - where it makes no sense. At the same time, I think that field could be ambiguous - so that we will miss it in this map. But they are useless for sonar python

Reply:
After discussion, we decided we will expose "type hints" of members in ClassSymbol, without exposing `InferredType`